Sources/UI/ActionButton/ActionButton.swift

```diff
@@ -27,43 +27,50 @@ public struct ActionButton: View {
     Button(action: {
       properties.actionHandler?()
     }) {
-      HStack {
-        properties.leftImage?.image
-          .resizable()
-          .aspectRatio(contentMode: .fit)
-          .frame(width: properties.leftImage?.size.width,
-                 height: properties.leftImage?.size.height)
+      VStack {
         Spacer()
-        HStack {
-          properties.titleLeftImage?.image
+          .frame(minHeight: 1)
+        HStack{
+          properties.leftImage?.image
             .resizable()
             .aspectRatio(contentMode: .fit)
-            .frame(width: properties.titleLeftImage?.size.width,
-                   height: properties.titleLeftImage?.size.height)
-          Text(properties.title)
-            .font(properties.font)
-          properties.titleRightImage?.image
+            .frame(width: properties.leftImage?.size.width,
+                   height: properties.leftImage?.size.height)
+          Spacer()
+          HStack {
+            properties.titleLeftImage?.image
+              .resizable()
+              .aspectRatio(contentMode: .fit)
+              .frame(width: properties.titleLeftImage?.size.width,
+                     height: properties.titleLeftImage?.size.height)
+            Text(title)
+              .font(properties.font)
+            properties.titleRightImage?.image
+              .resizable()
+              .aspectRatio(contentMode: .fit)
+              .frame(width: properties.titleRightImage?.size.width,
+                     height: properties.titleRightImage?.size.height)
+          }
+          
+          Spacer()
+          properties.rightImage?.image
             .resizable()
             .aspectRatio(contentMode: .fit)
-            .frame(width: properties.titleRightImage?.size.width,
-                   height: properties.titleRightImage?.size.height)
+            .frame(width: properties.rightImage?.size.width,
+                   height: properties.rightImage?.size.height)
+            .padding(.trailing, 10)
         }
+        
         Spacer()
-        properties.rightImage?.image
-          .resizable()
-          .aspectRatio(contentMode: .fit)
-          .frame(width: properties.rightImage?.size.width,
-                 height: properties.rightImage?.size.height)
+          .frame(minHeight: 1)
```

Comment:
Is that ok to have 1 here?

Reply:
If we go without it Spacer will use a default value which will be greater than 1. That won't allow us to go with a lower height than that default value. I don't think we'll need super-thin buttons but still I just didn't want to have that constraint.